validator/node/node.go

```diff
@@ -263,6 +263,32 @@ func moveDb(cliCtx *cli.Context, accountsDir string) string {
 }
 
 func (s *ValidatorClient) initializeForWeb(cliCtx *cli.Context) error {
+	var keyManager keymanager.IKeymanager
+	var err error
+	// Read the wallet from the specified path.
+	w, err := wallet.OpenWalletOrElseCli(cliCtx, func(cliCtx *cli.Context) (*wallet.Wallet, error) {
+		return nil, nil
+	})
+	if err != nil {
+		return errors.Wrap(err, "could not open wallet")
+	}
+	if w != nil {
```

Comment:
If a user already has a wallet, we shouldn't block the validator client from starting if using --web